Examples/StylesAndOptions/Example/NavigationExample.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  NavigationExample.swift
+//  Example
+//
+//  Created by Mayur Deshmukh on 23/10/18.
+//  Copyright © 2018 iZettle. All rights reserved.
+//
+
+import Foundation
+import Presentation
+import Flow
+
+struct NavigationExample { }
+
+struct TestNavigationBarHiding {
+
+}
+
+extension TestNavigationBarHiding: Presentable {
+    func materialize() -> (UIViewController, Disposable) {
+        let nc = UINavigationController()
+
+        let bag = DisposeBag()
+        bag += nc.present(Presentation(NavigationExample())).onValue {
```

Comment:
flatMapLatest will cause problems as it will signal error when we go back. We need `flatMapIgnoringErrors` here. @iZNunev has already worked on it. But it needs to be merged in Flow.